modules/docs/mdx/9.0-UPGRADE-GUIDE.mdx

```diff
@@ -52,133 +86,174 @@ handled by the codemod are marked with 🤖 in the Upgrade Guide.**
 the changes from the codemod as a single isolated commit (separate from other changes) so you can
 roll back more easily if necessary.
 
-## Component Deprecations
+## Token Updates
+
+### Depth
 
-### Deprecation Types
+**PR:** [#2091](https://github.com/Workday/canvas-kit/pull/2091)
```

Comment:
I've added links to corresponding PRs for each section in this upgrade guide. I found it helpful to refer to the PRs as I was reviewing content in the guide; I think they'll provide helpful context to consumers of the guide as well.

I originally linked the PRs within the headings themselves (similar to a squashed PR commit):

![image](https://user-images.githubusercontent.com/146020/229940715-6ab5260d-da18-4534-bb9e-9faccd2baa55.png)

I liked the conciseness of this approach, but it resulted in strange heading IDs such as `depth-2091` so I opted for listing the PR below the headings.

Reply:
I like this cause there could be multiple PRs per section heading, so listening them below makes more sense to me!

Reply:
I wonder if we could link to either the PR or the changelog

Reply:
Looking at the [changelog for v8](https://github.com/Workday/canvas-kit/blob/master/CHANGELOG.md#v801-2022-11-02) (technically 8.0.1), it feels like linking to the PR would be cleaner. The changelog has a lot of noise from other PRs, plus you can't link to a specific PR line item in the changelog.

Unless I'm misunderstanding what you're suggesting by linking to the changelog.

Reply:
makes sense to me!